regression/basic_flow.yaml

```diff
@@ -0,0 +1,32 @@
+name: streaming-regression-test-basic-flow
```

Comment:
Sorry, I did not think a proper name when I created a task. Can we rename this file to `iterate_data.yaml` or something better ?